pkg/apperrors/errors.go

```diff
@@ -242,7 +242,7 @@ type ResourceNotFoundError struct {
 // Error provides the error as a string
 func (e *ResourceNotFoundError) Error() string {
 	return fmt.Sprintf(
-		"Could not not find resource %T with error: %v",
+		"Could not find resource %T with error: %v",
```

Comment:
Changed in https://github.com/CMSgov/easi-app/pull/1398/commits/9be9c5575ddbff24b5210fe03de3c3dcbb16b91d to test that the method returns a `ResourceNotFoundError` type, instead of checking the exact error message. I don't see any failing test in `pkg/graph/schema.resolvers_accessibility_request_test.go`.